### DISCUSSION TITLE
Install Jellyfin 1.6.1 to your Roku

### DISCUSSION THREAD

**Discussion by 1hitsong** [jellyfin-archive/jellyfin-roku-legacy#897] [General]
# Steps
### Step 1 
Put Roku in Developer mode
### Step 2
Access the Development Application Installer

Both of these steps are detailed in Roku's official documentation: https://blog.roku.com/developer/developer-setup-guide
**You do not need to enroll in the dev program!**

### Step 3 
Upload ZIP file to Roku

Download [Jellyfin-Roku 1.6.1](https://github.com/jellyfin/jellyfin-roku/releases/download/v1.6.1/Jellyfin_Roku-dev.zip).

From the Development Application Installer screen, click Upload  button, select the downloaded Jellyfin_Roku-dev.zip file.

This will install Version 1.6.1 on your device. It will not replace version 1.6.2, they will be installed side-by-side.

**Comment by cewert** [comment]
These instructions are now on the readme. Closing.
